ios/Scripts/generate_app_icons_from_source.py

```
@@ -0,0 +1,111 @@
+
+    # Проверяем наличие PIL
+    try:
+        from PIL import Image
+    except ImportError:
+        print("❌ Требуется библиотека Pillow:")
+        print("pip install Pillow")
+        sys.exit(1)
+
+    # Генерируем иконки
+    if generate_all_icons_from_source(args.source):
+        print("\n🎉 Все иконки успешно созданы!")
```

Ruff (0.13.1)

String contains ambiguous `В` (CYRILLIC CAPITAL LETTER VE). Did you mean `B` (LATIN CAPITAL LETTER B)?

(RUF001)